src/wallet/wallet.h

```diff
@@ -7,6 +7,7 @@
 #define BITCOIN_WALLET_WALLET_H
 
 #include "amount.h"
+#include "main.h"
```

Comment:
Great! Could you try to remove it then?  I rmember not being able to do it in the past due to the many functions/methods defined in the .h instead of the .cpp but hopefully that has already changed.